honeybee_vtk/write.py

```diff
@@ -1,21 +1,16 @@
-"""Write vtk files to the disk."""
```

Comment:
`writer` might be a better name for this module and the function itself. `from writer import writer`